docs/testrunner-toolkit/typescript.md

```diff
@@ -58,22 +57,28 @@ Except for the TestCafe image, these `.ts` files cannot run directly on any Test
    >
    > For more information on how to properly configure `tsconfig.json` please visit the [documentation](https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html#writing-a-configuration-file).
 
-4. Next, edit the `files` and `suites` fields in `.sauce/config.yml` in order to ignore the `.ts` files and instead place the `.js` files inside the Testrunner Toolkit container:
+4. Next, edit the `suites` fields in `.sauce/config.yml` in order to ignore the `.ts` files and instead place the `.js` files inside the Testrunner Toolkit container:
     
    ```yaml
-   files:
-     - tests/
    suites:
      - name: "basic test"
-       match: ".*.(spec|test).js"
+       testMatch: 'tests/*.js'
    ```
    
    By default `saucectl` will pickup any `.js` files located in the designated directory, however with the `suites` field you can set more granular control with regular expressions.
 
 5. Finally, run `saucectl` to execute your TypeScript tests:
```

Comment:
This statement seems a bit confusing to me....aren't you technically running the JavaScript tests you just transpiled, NOT the Typescript tests?

Reply:
You are right, we are running JavaScript tests. I left this part as it was before. How do to suggest rewriting it?